frontend/src/components/object/ObjectFileDetails.vue

```diff
@@ -2,25 +2,42 @@
 import { onMounted, ref } from 'vue';
 import { useRoute } from 'vue-router';
 import { storeToRefs } from 'pinia';
+import Button from 'primevue/button';
```

Comment:
Style: We'll want to reorder the imports so that the pinia one comes before the vue one.